Fix gcc-4.1 warning:
dereferencing type-punned pointer will break strict-aliasing rules

--- src/convert-dg-to-py.cc~	2011-06-21 10:41:18.000000000 +0200
+++ src/convert-dg-to-py.cc	2011-10-31 15:14:17.000000000 +0100
@@ -73,7 +73,7 @@
 	    throw ExceptionPython(ExceptionPython::VALUE_PARSING,
 				   "bool");
 	  }
-	  bvalue = (pyObject == Py_True);
+	  bvalue = PyObject_IsTrue(pyObject);
 	  return Value(bvalue);
 	  break;
 	case (Value::UNSIGNED) :
